libskale/State.cpp

```diff
@@ -321,6 +321,39 @@ void State::safeRemoveAllPartialTransactionReceipts() {
 }
 
 
+void State::safeRemoveLegacyPartialTransactionReceipts() {
+    if ( m_db_ptr ) {
+        m_db_ptr->cleanupLegacyTransactionReceipts();
+    }
+}
+
+
+dev::eth::TransactionReceipts State::safeLegacyPartialTransactionReceipts() {
+    dev::eth::TransactionReceipts legacyPartialTransactionReceipts;
+    if ( m_db_ptr ) {
+        auto rawTransactionReceipts = m_db_ptr->getLegacyPartialTransactionReceipts();
+        if ( !rawTransactionReceipts.empty() ) {
+            dev::RLP rlp( rawTransactionReceipts );
+            dev::eth::BlockReceipts blockReceipts( rlp );
+            legacyPartialTransactionReceipts.insert( legacyPartialTransactionReceipts.end(),
```

Comment:
you can simply do `return blockReceipts.receipt;`

Reply:
addressed